charts/sourcegraph-executor/templates/executor/executor.Deployment.yaml

```diff
@@ -0,0 +1,150 @@
+{{- if .Values.executor.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    description: Runs sourcegraph executors
+    kubectl.kubernetes.io/default-container: executor
+  labels:
+    {{- include "sourcegraph.labels" . | nindent 4 }}
+    {{- if .Values.executor.labels }}
+      {{- toYaml .Values.executor.labels | nindent 4 }}
+    {{- end }}
+    deploy: sourcegraph
+    app.kubernetes.io/component: executor
+spec:
+  selector:
+    matchLabels:
+      {{- include "sourcegraph.selectorLabels" . | nindent 6 }}
+      app: executor
+  minReadySeconds: 10
+  replicas: {{ .Values.executor.replicaCount }}
+  revisionHistoryLimit: 10
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: RollingUpdate
+  template:
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/default-container: executor
+      {{- if .Values.sourcegraph.podAnnotations }}
+      {{- toYaml .Values.sourcegraph.podAnnotations | nindent 8 }}
+      {{- end }}
+      {{- if .Values.executor.podAnnotations }}
+      {{- toYaml .Values.executor.podAnnotations | nindent 8 }}
+      {{- end }}
+      labels:
+      {{- include "sourcegraph.selectorLabels" . | nindent 8 }}
+      {{- if .Values.sourcegraph.podLabels }}
+      {{- toYaml .Values.sourcegraph.podLabels | nindent 8 }}
+      {{- end }}
+      {{- if .Values.executor.podLabels }}
+      {{- toYaml .Values.executor.podLabels | nindent 8 }}
+      {{- end }}
+        deploy: sourcegraph
+        app: executor
+    spec:
+      containers:
+        - name: executor
+          image: {{ include "sourcegraph.image" (list . "executor") }}
+          imagePullPolicy: {{ .Values.sourcegraph.image.pullPolicy }}
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: debug
+              scheme: HTTP
+            initialDelaySeconds: 60
+            timeoutSeconds: 5
+          readinessProbe:
+            httpGet:
+              path: /ready
+              port: debug
+              scheme: HTTP
+            periodSeconds: 5
+            timeoutSeconds: 5
+          ports:
+            - containerPort: 6060
+              name: debug
+          terminationMessagePolicy: FallbackToLogsOnError
+          env:
+          {{- range $name, $item := .Values.executor.env}}
+          - name: {{ $name }}
+            {{- $item | toYaml | nindent 10 }}
+          {{- end }}
```

Comment:
This is indented 2 spaces fewer than the static env items below, is that intentional?